app/models/grouping.rb

```diff
@@ -264,7 +264,7 @@ def add_member(user, set_membership_status = StudentMembership::STATUSES[:accept
     if user.has_accepted_grouping_for?(self.assessment_id) || user.hidden
       nil
     else
-      member = StudentMembership.new(user: user, membership_status:
```

Comment:
change method parameter from `user` to `role`

(sorry I can't comment on the line of the function definition -- line 263)